addons/test_mass_mailing/tests/test_performance.py

```diff
@@ -51,7 +51,7 @@ def test_send_mailing(self):
         })
 
         # runbot needs +50 compared to local
-        with self.assertQueryCount(__system__=1714, marketing=1715):
+        with self.assertQueryCount(__system__=1715, marketing=1716):
```

Comment:
@tde-banana-odoo 
do you agree with this change? I didn't check what it's one more query